### ISSUE TITLE
Extention crashes on Windows (BUG)

### ISSUE THREAD

**Issue by hemangajmera** [camel-tooling/camel-lsp-client-vscode#472]
# Issue information
I am repeatedly gets this exception and it crashes

```
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:107)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:58)
	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:88)
Caused by: java.lang.NullPointerException
	at java.base/java.io.Reader.<init>(Reader.java:167)
	at java.base/java.io.InputStreamReader.<init>(InputStreamReader.java:72)
	at org.apache.camel.kafkaconnector.catalog.CamelKafkaConnectorCatalog.initCatalog(CamelKafkaConnectorCatalog.java:65)
	at org.apache.camel.kafkaconnector.catalog.CamelKafkaConnectorCatalog.<init>(CamelKafkaConnectorCatalog.java:52)
	at com.github.cameltooling.lsp.internal.catalog.util.CamelKafkaConnectorCatalogManager.<init>(CamelKafkaConnectorCatalogManager.java:23)
	at com.github.cameltooling.lsp.internal.CamelTextDocumentService.<init>(CamelTextDocumentService.java:94)
	at com.github.cameltooling.lsp.internal.CamelLanguageServer.<init>(CamelLanguageServer.java:53)
	at com.github.cameltooling.lsp.internal.Runner.main(Runner.java:65)
	... 8 more
[Info  - 11:24:59 am] Connection to server got closed. Server will restart.
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:107)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:58)
	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:88)
Caused by: java.lang.NullPointerException
	at java.base/java.io.Reader.<init>(Reader.java:167)
	at java.base/java.io.InputStreamReader.<init>(InputStreamReader.java:72)
	at org.apache.camel.kafkaconnector.catalog.CamelKafkaConnectorCatalog.initCatalog(CamelKafkaConnectorCatalog.java:65)
	at org.apache.camel.kafkaconnector.catalog.CamelKafkaConnectorCatalog.<init>(CamelKafkaConnectorCatalog.java:52)
	at com.github.cameltooling.lsp.internal.catalog.util.CamelKafkaConnectorCatalogManager.<init>(CamelKafkaConnectorCatalogManager.java:23)
	at com.github.cameltooling.lsp.internal.CamelTextDocumentService.<init>(CamelTextDocumentService.java:94)
	at com.github.cameltooling.lsp.internal.CamelLanguageServer.<init>(CamelLanguageServer.java:53)
	at com.github.cameltooling.lsp.internal.Runner.main(Runner.java:65)
	... 8 more
[Error - 11:25:01 am] Connection to server got closed. Server will not be restarted.
```

> Define issue title **short** and **most specific** to the problem.

> Tag each new issue with relevant **LABELS** (*Bug*, *Improvement*, etc.).

### Environment

- Language Support for Apache Camel version: e.g. "**v0.0.13**" ?
v0.0.28
- OS:
    - [ ] Linux (*RHEL*, *Fedora*, *Ubuntu*, ...)
    - [ X] Windows
    - [ ] macOS

### Description of wrong behavior

- ...

### Exact steps to reproduce issue

1. Opening camel project with maven
2. ...
3. ...

### Description
These are dependencies in pom.xml

```
<dependencies>
        <dependency>
            <groupId>org.apache.camel.springboot</groupId>
            <artifactId>camel-spring-boot-starter</artifactId>
        </dependency>
        <dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-configuration-processor</artifactId>
        </dependency>        
        <dependency>
            <groupId>org.apache.camel.springboot</groupId>
            <artifactId>camel-kafka-starter</artifactId>
        </dependency>
    </dependencies>   
```
Camel version is 3.4.4
- ...

### Expected behaviour

- There should be no error

### Screenshot (*optional*)

- ...

**Comment by oscerd** [comment]
The dependencies reported here, have nothing to do with camel-kafka-connector.

**Comment by apupier** [comment]
> The dependencies reported here, have nothing to do with camel-kafka-connector.

agree. But the Camel Language Server is initializing the Camel Kafka Connector Catalog as soon as it is starting. It allows to have a faster completion/hover when it is Camel Kafka Connector related.

**Comment by hemangajmera** [comment]
Let me know if you need more information about the error. Camel LSP used to work on windows correctly earlier. This might be crashing due to some recent change.

**Comment by apupier** [comment]
@hemangajmera thanks for your report, it is greatly appreciated. It contains all information needed.
The root cause has been fixed https://github.com/apache/camel-kafka-connector/issues/697 . You will need to wait that the fix bubbles up in the VS Code extension.

**Comment by apupier** [comment]
the fix is bubbling up.
using latest vsix from this snapshot repository should fix the issue https://download.jboss.org/jbosstools/vscode/snapshots/vscode-apache-camel/

a release with the fix should occur this week.

To avoid this kind of regression on Windows, windows system is in progress to be added in the Continuous Integration https://app.circleci.com/pipelines/github/camel-tooling/camel-language-server/93/workflows/0606301f-c8c3-422f-8e05-a814a833a903/jobs/307

sorry for this regression

**Comment by apupier** [comment]
fix is available in release 0.0.29
